website/source/api/secret/identity/groups.html.md

```diff
@@ -0,0 +1,228 @@
+---
+layout: "api"
+page_title: "Identity Secret Backend - HTTP API"
+sidebar_current: "docs-http-secret-identity-groups"
```

Comment:
Change the filename to `identity/group.html.md` and the ref docs-http-secret-identity-group

Reply:
@calvn Good suggestion. Fixed this for all (entities -> entity, groups -> group, entity-aliases -> entity-alias, group-aliases -> group-alias).